ansible-ipi-install/roles/installer/tasks/59_cleanup_bootstrap.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Get list of all VMs
+  virt:
+    command: list_vms
+  register: all_vms
+  become: yes
+
+- name: Get list of all running VMs
+  virt:
+    command: list_vms
+    state: running
+  register: running_vms
+  become: yes
+
+- name: Destroy old bootstrap VMs, if any
+  virt:
+    name: "{{ item }}"
+    command: destroy
+  with_items: "{{ running_vms.list_vms }}"
+  when: item is search("bootstrap")
+  become: yes
+
+- name: Undefine old bootstrap VMs, if any
+  virt:
+    name: "{{ item }}"
+    command: undefine
+  with_items: "{{ all_vms.list_vms }}"
+  when: item is search("bootstrap")
+  become: yes
+
+- name: Find old bootstrap VM Storage
+  find:
+    paths: "{{ default_libvirt_pool_dir }}"
+    patterns: '*-bootstrap,*-bootstrap.ign'
```

Comment:
I'm not sure if this is still valid but back in the day there were three volumes, those two + `*-bootstrap-orig`

Reply:
I haven't seen the `-bootstrap-orig` in recent builds.